lib/declare_schema/model/habtm_model_shim.rb

```diff
@@ -66,8 +69,8 @@ def ignore_indexes
 
       def constraint_specs
         [
-          ForeignKeyDefinition.new(self, foreign_keys.first, parent_table: foreign_key_classes.first.table_name, constraint_name: "#{join_table}_FK1", dependent: :delete),
-          ForeignKeyDefinition.new(self, foreign_keys.last, parent_table: foreign_key_classes.last.table_name, constraint_name: "#{join_table}_FK2", dependent: :delete)
+          ForeignKeyDefinition.new(self, foreign_keys.first, parent_table: foreign_key_classes.first.table_name, constraint_name: "#{join_table}_FK1"),
+          ForeignKeyDefinition.new(self, foreign_keys.last, parent_table: foreign_key_classes.last.table_name, constraint_name: "#{join_table}_FK2")
```

Comment:
I'm under the impression that the we do **not** want to delete the associations because they can belong to other objects

Reply:
I think we do still want the dependent delete put in. That way if one of the linked objects is deleted, we don't have the join record hanging around pointing to nothing.

Reply:
Oh gotchya, I'll go ahead and undo this